docker-compose.yml

```diff
@@ -22,15 +25,16 @@ services:
     depends_on:
       - mongo
     environment:
-      MONGO_URL: "mongodb://mongo:27017/reaction"
-      MONGO_OPLOG_URL: "mongodb://mongo:27017/local"
+      MONGO_URL: "mongodb://mongo.streams.reaction.localhost:27017/reaction"
+      MONGO_OPLOG_URL: "mongodb://mongo.streams.reaction.localhost:27017/local"
```

Comment:
These URLS could be left unchanged. Mongo is still on the default network Better if the network addition doesn't bleed into this projects internals?

Reply:
Testing now...

Reply:
OK I made those changes and things still work. Pushed amended commit.